docs/guides/langchain.md

```diff
@@ -42,23 +42,31 @@ spec:
         template:
             spec:
                 containers:
-                    - name: "ai-model-7b"
-                      image: modelzai/llm-llama-7b:latest # GPU requirements: A100(40GB)
-                      # alternatively use, modelzai/llm-bloomz-560m:latest (even works on CPU)
+                    - name: "ai-model"
+                      image: swarnimarun/llama-server:latest-cuda
+                      # GPU requirements: T4(16GB)
+                      # For CPU : "swarnimarun/llama-server:latest" - 16GB
+                      args:
+                        - "-m"
+                        - "lmstudio-community/Meta-Llama-3-8B-Instruct-GGUF"
+                        - "-g"
+                        - "33"
+                        - "-q"
+                        - "q8"
         accelerator:
             interface: "CUDA"
             minVersion:
                 major: 7
         resources:
             limits:
                 cpu: "1"
-                memory: "16Gi" # a decent amount of RAM is required for loading the model as well
+                memory: "2Gi" # some amount of RAM is required for loading the model as well, for cpu use atleast 16GB of RAM
```

Comment:
I'd always assume we need more RAM than the model size. Although it could be streamed onto the GPU; a slight changed could result in OOM.

Reply:
We load data using mmap in llama so it should load it correctly even with less ram available.

Reply:
Ah right so it maps the file and does stream it into GPU memory. Still I just wouldn't recommend limiting the RAM below the model size unless you have some specific information about your environment that makes it necessary/beneficial. It could result in better or worse performance, but the thing that concerns me is that it puts extra pressure on the kernel memory management to perform correctly.

You could note that limiting the RAM may result in less RAM usage which may benefit other services?